packages/openapi-spec/src/openapi-spec-v2.ts

```diff
@@ -1037,5 +1037,6 @@ export function createEmptyApiSpec(): OpenApiSpec {
       version: '1.0.0',
     },
     paths: {},
+    definitions: {},
```

Comment:
The property `definitions` is optional according to [Swagger Spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#fixed-fields). Please leave it out from the empty API spec. (It will save you updates of so many tests below.)